src/resolvers/Mutation.js

```diff
@@ -192,11 +192,82 @@ async function _createTags(tags_contactNumbers, context) {
   return { create: arr };
 }
 
+async function deletePost(parent, args, context, info) {
+  const id = args.id;
```

Comment:
We need protection, you can only do this to your own.
So add it with 
userId = context.userId
and 
activegroup = context.activeGroup